shell.nix

```diff
@@ -23,9 +23,14 @@ let
     (pkgs.writeScriptBin "install-editor" ''
       #!/usr/bin/env bash
       set -e
-      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/utopia-vscode-common
+      update-vscode-build-extension
+      cd $(${pkgs.git}/bin/git rev-parse --show-toplevel)/editor
       ${node}/bin/npm --scripts-prepend-node-path=true install
-      ${node}/bin/npm --scripts-prepend-node-path=true run build
+    '')
+    (pkgs.writeScriptBin "install-editor-ci" ''
```

Comment:
I think it's important information that for first time checkout out the editor, this is the script to run.

Maybe we could make a dedicated `install-editor-first-time` or a more general `setup` script that would also execute on the CI?